athena-api-boot/src/main/java/org/catools/athena/rest/core/controller/AthenaUserController.java

```diff
@@ -41,46 +40,62 @@ public AthenaUserController(AthenaCoreService athenaCoreService) {
     public ResponseEntity<Set<UserDto>> getUsers() {
         final Set<UserDto> users = athenaCoreService.getUsers();
         return users.isEmpty() ?
-                ResponseEntity.noContent().cacheControl(CACHE_MAX_AGE_ONE_HOUR).build() :
-                ResponseEntity.ok().cacheControl(CACHE_MAX_AGE_ONE_HOUR).body(users);
+                ResponseEntityUtils.noContent() :
+                ResponseEntityUtils.ok(users);
     }
 
-    @GetMapping(USER)
+    @GetMapping(USER_PATH)
     @Operation(
             summary = "Retrieve user by name",
             responses = {
                     @ApiResponse(responseCode = "200", description = "Successfully returned data"),
                     @ApiResponse(responseCode = "204", description = "No content to return")
```

Comment:
I would add 400 for the case if username is not provided

Reply:
Also I believe that we should handle cases when username is not provided. I can do it in a separate PR if you want me to work on it.

Reply:
Same for other parameters we send to methods get methods in services and controllers. Probably we need to talk about where we want to handle this.

Reply:
Agree, lets do it in a separate PR, and sure, appreciate if you take care of it